server/src/metakb/schemas/api.py

```diff
@@ -8,11 +9,78 @@
     VariantTherapeuticResponseStudyStatement,
 )
 from ga4gh.va_spec.base import Statement
+from ga4gh.vrs import VRS_VERSION
+from ga4gh.vrs import (
+    __version__ as vrs_python_version,
+)
 from pydantic import BaseModel, ConfigDict, StrictStr
 
 from metakb import __version__
 
 
+class ServiceEnvironment(str, Enum):
+    """Define current runtime environment."""
+
+    DEV = "dev"
+    PROD = "prod"
+    TEST = "test"
+    STAGING = "staging"
+
+
+class ServiceOrganization(BaseModel):
+    """Define service_info response for organization field"""
+
+    name: Literal["Variant Interpretation for Cancer Consortium"] = (
+        "Variant Interpretation for Cancer Consortium"
+    )
+    url: Literal["https://cancervariants.org/"] = "https://cancervariants.org/"
+
+
+class ServiceType(BaseModel):
+    """Define service_info response for type field"""
+
+    group: Literal["org.cancervariants"] = "org.cancervariants"
+    artifact: Literal["MetaKB API"] = "MetaKB API"
+    version: str = __version__
+
+
+class SpecMetadata(BaseModel):
+    """Define substructure for reporting specification metadata."""
+
+    vrs_version: str = VRS_VERSION
+
+
+class ImplMetadata(BaseModel):
+    """Define substructure for reporting metadata about internal software dependencies."""
+
+    vrs_python_version: str = vrs_python_version
```

Comment:
We should also include cat-vrs/cat-vrs-python and va-spec/va-spec-python (which made me realize, I forgot to bump this in the latest va-spec-python release)

Reply:
Oh JK. I just saw the PR description.